src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss

```diff
@@ -0,0 +1,61 @@
+@import "vars";
+
+//
+// ===== Mixins for <hx-checkbox> facade states =====
+//
+
+// ========== PRISTINE ==========
+@mixin __checkboxFacade($psuedo-state: null) {
+  @if $psuedo-state == hover {
+    background-color: $cyan-50;
+    border-color: $cyan-500;
+    color: $cyan-500;
+  }
+}
+
+// ========== INVALID ==========
+@mixin __checkboxFacade--invalid($psuedo-state: null) {
+  @if $psuedo-state == focus {
+    border-color: $red-900;
+    box-shadow: $focus-glow-invalid;
+  }
+  @else if $psuedo-state == hover {
+    background-color: $red-100;
+    border: 2px solid $red-900;
+    color: $red-900;
+
+    &:hover {
+      background-color: $red-100;
+      border-color: $red-900;
+      color: $red-900;
+    }
+  }
+  @else {
+    border: 2px solid $red-900;
+    color: $red-900;
+  }
+}
+
+// ========== DISABLED ==========
+@mixin __checkboxFacade--disabled($psuedo-state: null) {
+  @if $psuedo-state == focus {
+    border-color: $gray-500;
+  }
+  @else if $psuedo-state == hover {
+    background-color: $gray-100;
+    border-color: $gray-500;
+    color: $gray-500;
+  }
+}
+
+@mixin hxCheckboxFacade($state: null, $psuedo-state: null) {
```

Comment:
Recommend splitting `$psuedo-state` (misspelled) into `$focused: false` and `$hovered: false`.

Reply:
Misspelling fixed.